keys/employees-keys/README.md

```diff
@@ -335,8 +303,117 @@ but before you can do that your key with new expiry date have to be
 In the second case, you should revoke the key. The procedure is
 [here](https://superuser.com/a/1526287).
 
-#### If we extend expiry every time, why not set "key does not expire"? 
+### If we extend expiry every time, why not set "key does not expire"? 
 
 Because people forget about keys or stop using them for many reasons. If
 someone extends expiry time, it means someone cares, and the key is still in
 use.
+
+### What to do when primary/master private key was lost, compromised or will no longer be used? 
+
+Following guide was made for:
+- recovery in case of loosing access to you primary/master private key,
+- evidence of primary/master key compromise,
+- event when employee of 3mdeb finished his/her cooperation and should no loger
+  sign 3mdeb related development work.
+
+Guide is made with assumption, that you still have access to:
+- private key or at least previously generated revocation certificate - please
+  note that gpg save revocation certificate automatically on machine you
+  generated key in `/home/USERNAME/.gnupg/openpgp-revocs.d/KEYID.rev`
+- list of servers to which your key was uploaded - in theory servers should
+  sync between each other,
+- list of keys you signed - check below how to obtain it,
+
+Correct key management would require use of subkeys and private part of your
+master key should never be lost because it should stored in safe location.
+
+#### Inform your Team Leader or Manager
+
+Please immiedietly let know your 3mdeb Team Leader or Manager about potential
+problem with your GPG keys. Delaying this process you put 3mdeb at risk of
+compromise.
+
+#### Generate revocation certificate
+
+If you don't have revocation certificate, but have access to private key please
+generate revocation certificate by using:
+
+```shell
+gpg --output revoke.asc --gen-revoke KEYID
```

Comment:
```suggestion
gpg --output your.name@3mdeb.com.rev --gen-revoke KEYID
```

Reply:
It asks me for a reason here:

```
Create a revocation certificate for this key? (y/N) y
Please select the reason for the revocation:
  0 = No reason specified
  1 = Key has been compromised
  2 = Key is superseded
  3 = Key is no longer used
  Q = Cancel
(Probably you want to select 1 here)
Your decision? 
```

This command only generates certificate, not actually revokes the key, right?

I assume we should gather these from current employees?

Reply:
> This command only generates certificate, not actually revokes the key, right?

@macpijan Yes, it generates a revocation certificate, which can then be applied to the public key. At least, this is how I understand it. We have two situations: either we know what the reason is, or we don't. A further case is our situation when we generate certificates for new employees. If something happens with the key, we know the reason and should use the correct one. We are always able to provide a description after selecting a reason.

> I assume we should gather these from current employees?

Yes.

Please let m know if this section need improvement.

Reply:
@macpijan ping?